.github/workflows/build.yml

```diff
@@ -165,6 +164,25 @@ jobs:
           MAKE: make -j2 --output-sync=recurse
           SAGE_NUM_THREADS: 2
 
+      # Testing
+
+      - name: Test changed files (sage -t --new)
+        run: |
+          # We run tests with "sage -t --new"; this only tests the uncommitted changes.
```

Comment:
`if: always() && steps.worktree.outcome == 'success'`?

Reply:
thanks, i've made a similar change in 8224cb0